src/sinks/blackhole/config.rs

```diff
@@ -19,13 +23,17 @@ pub struct BlackholeConfig {
     /// The number of seconds between reporting a summary of activity.
     ///
     /// Set to `0` to disable reporting.
-    #[derivative(Default(value = "1"))]
+    #[derivative(Default(value = "default_print_interval_secs()"))]
     #[serde(default = "default_print_interval_secs")]
-    pub print_interval_secs: u64,
+    #[serde_as(as = "serde_with::DurationSeconds<u64>")]
+    #[configurable(metadata(docs::examples = 10))]
+    pub print_interval_secs: Duration,
 
     /// The number of events, per second, that the sink is allowed to consume.
     ///
     /// By default, there is no limit.
+    #[configurable(metadata(docs::examples = 1000))]
+    #[configurable(metadata(docs::type_unit = "events per second"))]
     pub rate: Option<usize>,
```

Comment:
The description of this option, and the variable name are clashing for me....

It sounds less like a rate and more like a max / limit ?

If it was a rate/interval seems like we'd be assigning it a Duration.

Reply:
I ended up dropping the units since there isn't support for that rate. You are right in that it's more of a max rate.

Reply:
Fair, I mostly just pulled it from the previous description 👍

Reply:
Ah, so this unit could _probably_ be named "events" the `rate` is: `usize(events)/1(seconds)`